build/index.js

```diff
@@ -18,7 +18,7 @@ exports.onBeforeBuild = function (api, app, config, cb) {
              'var js, fjs = d.getElementsByTagName(s)[0];',
              'if (d.getElementById(id)) {return;}',
              'js = d.createElement(s); js.id = id;',
-             'js.src = "//connect.facebook.net/en_US/all.js";',
+             'js.src = "//connect.facebook.net/en_US/sdk.js";',
```

Comment:
I'll run some tests ASAP, am keen to switch over to to sdk.js as all.js was deprecated last month - I think the facebook servers are routing old v1 endpoints to v2 so they probably act very similarly. That said, all.js doesn't expose AppEvents which is the driving force behind the change.

I've addressed the other snags in my next commit, but will make another pull request once I've had the chance to see what the knock-on effect of this change is.